Makefile

```diff
@@ -115,22 +115,35 @@ etcd:
 					$(LD_FLAGS)\"" \
 		$(MAKE) -C etcd
 
-.PHONY: verify verify-images verify-assets licensecheck
-verify: verify-images verify-assets verify-sh verify-go verify-py verify-container licensecheck
+# Default verify target for developers
+.PHONY: verify
+verify: verify-fast
 
+# Fast verification checks that developers can/should run locally
+.PHONY: verify-fast
+verify-fast: verify-go verify-assets verify-sh verify-py
+
+# Full verification checks that should run in CI
+.PHONY: verify-ci
+verify-ci: verify-fast verify-images verify-govulncheck licensecheck verify-container
```

Comment:
Let's also drop the vulnerability checks from CI until we have time to address the reported issues.  It should probably fall under the verify-go target anyhow.

Reply:
fixed